handler/panos.go

```diff
@@ -96,14 +105,18 @@ func NewPanos(c map[string]interface{}) (*Panos, error) {
 		providerConf: conf,
 		adminUser:    username,
 		configPath:   configPath,
+		autoCommit:   autoCommit,
 	}, nil
 }
 
 // Do executes panos' out-of-band Commit API and calls next handler while passing
 // on relevant errors
 func (h *Panos) Do(prevErr error) error {
 	log.Printf("[INFO] (handler.panos) commit. host '%s'", h.providerConf.Hostname)
```

Comment:
It still calls through to the next handler, but is a noop for it's functionality. I was thinking it'd be better to know it was called and didn't commit (maybe you had it configured to use the handler but had auto-commit off) than skip the message. 

It does seem rather verbose for very limited gain though. After thinking about it more I'm leaning toward removing it. 

We should talk about this at some point. Maybe look into coming up with a simple guideline about when and what types of log entries we want. For example, we could say we only log on actions (not on skipping them) or something.

Reply:
I found this log helpful for me when I was developing handlers. Maybe change it to be TRACE level instead? Agree, I think it would be great to have a discussion on logging.

🤔 To your point, I'm realizing that this log is pretty ambiguous. I'm understanding your interpretation of this log as something like 'entering the commit code but may or may not do actual commit'. I had interpreted it as 'starting to commit now...'. Maybe one option could be to include the autoCommit value in this log?

Reply:
I thought about moving it into the commit() method w/ TRACE or DEBUG (as there is a commit successful log entry at DEBUG level). But wasn't really sure it added anything to the log entries already in commit().

Having it log with the value of autoCommit or moving the log to an `else` clause so it logs when autoCommit is off might be useful.

I guess for now I'll just update it to have the autoCommit value displayed. Something along the lines of...

    "[TRACE] (handler.panos) commit enabled. host foo."

Reply:
I made that change. Let me know if you think of something better. Thanks.

Reply:
I didn't realize there was already a successful commit log entry, sorry! In that case, I'm ok with deleting this log as well. Also happy to keep the log as you've updated though - the message is super clear. I'll leave it up to you. Thanks!